dockerfiles/builder/Dockerfile-builder

```diff
@@ -28,7 +28,7 @@ CMD ["scripts/strip-engine.sh"]
 
 ENV APP_ENV=production
 
-FROM node:18-buster-slim as runner
+FROM node:21-bookworm-slim as runner
```

Comment:
@barankyle @HexaField It does work just fine as far as I can see but there might be some edge cases. Don't know if this is 100% necesarry for my changes though I did see some bugs with `packages/client/scripts/populate-assetlinks.ts` and node18 with my changes